var/spack/repos/builtin/packages/py-outcome/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyOutcome(PythonPackage):
+    """FIXME: Put a proper description of your package here."""
```

Comment:
There is a broken audit:
```
1. Package 'py-outcome' contains boilerplate that need to be removed
    /Users/runner/work/spack/spack/var/spack/repos/builtin/packages/py-outcome/package.py:9: boilerplate needs to be removed: """FIXME: Put a proper description of your package here."""
```

Reply:
I added the description so this is fixed.